### DISCUSSION TITLE
Which shell is `sh`?

### DISCUSSION THREAD

**Discussion by Rudxain** [termux/termux-app#3313] [Q&A] [answered]
`type sh` prints
```
sh is /data/data/com.termux/files/usr/bin/sh
```

I can't find any info about this program on the web. And it doesn't accept any flags/options (no `-` prefix allowed), so `--help` and `--version` don't work.

I need to know, because I have a script that _should_ be POSIX-compliant (I haven't used ShellCheck to test it, yet), and it works on Android's MirBSD Korn Shell (`mksh`): `/system/bin/sh`

**Comment by trygveaa** [comment, marked as answer]
```
~ $ dpkg -S $(which sh)
dash: /data/data/com.termux/files/usr/bin/sh
```

So it's [Dash](https://en.wikipedia.org/wiki/Almquist_shell#dash), same as in Debian and Ubuntu.
